walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html

```diff
@@ -9,21 +9,51 @@
     <div id="content">
       <h1>Laura's Image Gallery</h1>
       <div class="image-thumbnails">
-        <a href="images-me/laura-1.jpg"><img src="images-me/laura-1.jpg"/></a>
-        <a href="images-me/laura-2.jpg"><img src="images-me/laura-2.jpg"/></a>
-        <a href="images-me/laura-3.jpg"><img src="images-me/laura-3.jpg"/></a>
-        <a href="images-me/laura-4.jpg"><img src="images-me/laura-4.jpg"/></a>
-        <a href="images-me/laura-5.jpg"><img src="images-me/laura-5.jpg"/></a>
-        <a href="images-me/laura-6.jpg"><img src="images-me/laura-6.jpg"/></a>
-        <a href="images-me/laura-7.jpg"><img src="images-me/laura-7.jpg"/></a>
-        <a href="images-me/laura-8.jpg"><img src="images-me/laura-8.jpg"/></a>
-        <a href="images-me/laura-9.jpg"><img src="images-me/laura-9.jpg"/></a>
-        <a href="images-me/laura-10.jpg"><img src="images-me/laura-10.jpg"/></a>
-        <a href="images-me/laura-11.jpg"><img src="images-me/laura-11.jpg"/></a>
-        <a href="images-me/laura-12.jpg"><img src="images-me/laura-12.jpg"/></a>
-        <a href="images-me/laura-13.jpg"><img src="images-me/laura-13.jpg"/></a>
-        <a href="images-me/laura-14.jpg"><img src="images-me/laura-14.jpg"/></a>
-	<a href="images-me/laura-15.jpg"><img src="images-me/laura-15.jpg"/></a>
+        <a href="images-me/laura-1.jpg">
```

Comment:
So is the goal of the <a> tag here to make the image clickable to show by itself as well as displaying it normally?

Reply:
I believe so yes! I wanted to both display the image and make it clickable so you could see the whole picture. Was that what you were referring to? Thank you for the comments!

Reply:
Yup, sounds good